bin/MessagesBundle_fr.properties

```diff
@@ -115,6 +115,22 @@ Restart=Démarrage
 Drawing=Dessin
 FidoCad=Extensions FidoCadJ
 pcbline_width = Largeur piste C.I. :
+
+Theme_management=Theme and color management
```

Comment:
Put all the untranslated resources at the end of the file. They are easier to find for those who would like to work on them. (PS I will translate them, so it is not a big deal, but that keeps the order approximately the same among files)